### PR TITLE
add delegate's Name in 'ioctl node delegate' output 2169

### PR DESCRIPTION
**Pull request by lzxm160**
work on #2169

ioctl node delegate output after fairbank:

```
Epoch: 4808,  Start block height: 3360961,Total blocks produced in epoch: 275

Address                                     Name           Rank   Alias   Status   Blocks   ProbatedStatus    Votes
io14jp9s6mp85g2x00spq66cmdds4qe2ge5r0p72d   robotbp00021      1           active   12                       13752651.268634533599636965
io1s3z2pr47u47p69zn7phshgenckw6tr70tex80k   robotbp00018      2           active   11                       2289700.876322665986934979
io1eckfc699y4shqkc9ms8ts9sa254e5vtqyazkar   robotbp00020      3           active   11                       2289609.296529794438734839
io1dh8uwmva79vhk8mec998f7qgfud7jyz9v8lede   robotbp00013      4           active   11                       2289495.966536115897837167
io1gakgrsnsxmg9ed0tlcc2ukxzxdg6v8z2glw82e   robotbp00012      5           active   12                       2289494.821788705003484665
io1dg65erd07hrvyme0493f2kqj2utuvpyf6jeuhd   robotbp00001      6           active   12                       2289494.821788705003484665
io1se72frs5kn0j6gh0g3fk2usngu48sh2wjt4qgq   robotbp00004      7           active   12                       2289494.821788705003484665
io1pgth3m3e4klanwdrfrgk6yelw03ws64ml8c22y   robotbp00022      8           active   11                       2289494.821788705003484665
io1rm223d7jyflpg0cvkja4zmgnjz8hrrdgm09pu0   robotbp00000      9           active   12                       2289494.821788705003484665
io15ccuntztkkuu0xnfhqf2qt8y28kpu0mnr65050   robotbp00003     10           active   11                       2289494.821788705003484665
io1m8rge0h4xvmfzkc4yup766slcvn5cwymqeenpe   robotbp00014     11           active   12                       2289494.821788705003484665
io1etc9ffzsg7lvph3j4ncczg08llvame8k3asyzc   robotbp00015     12           active   12                       2289494.821788705003484665
io1343pwahz73s7vzd3l8jxxrjwcugxae2nx58zwf   robotbp00002     13           active   12                       2289494.821788705003484665
io1jhcm9au9u77l27pgkqe6kqdf58dh64d0k5kf2x   robotbp00008     14           active   12                       2289494.821788705003484665
io1az8tln8432h42vf54e0c0fmaqcytwm2nk82g95   robotbp00005     15           active   11                       2289494.821788705003484665
io1959ykdhkene09at5nfl42a7xjwf5wgplpg7eh9   robotbp00016     16           active   11                       2289494.821788705003484665
io1qendhfjmpmug6ywde0590lakvxp3ph869ehlyp   robotbp00007     17           active   11                       2289494.821788705003484665
io13n3382cjhaawmqfk4vmvvgllnryw4tf56qdtks   robotbp00011     18           active   11                       2289494.821788705003484665
io1e5uh4sfkgemds4fannncujcdylerdynddj3f43   robotbp00019     19           active   11                       2289494.821788705003484665
io1y4punwfvhxzee08a034922vwr48wz0p6ruy0ke   robotbp00009     20           active   11                       2289494.821788705003484665
io1vdw0q7dv82g3jhmapvze99fdxeyztjrxdp7wse   robotbp00006     21           active   11                       2289494.821788705003484665
io1c5a8kv4nljt4dy2mnxk23rj0255a9x4uvn4x5m   robotbp00010     22           active   11                       2289494.821788705003484665
io1j07xpufsj03sc0dg020jahacx2pnt6m8cdl6qt   robotbp00017     23           active   12                       2289494.821788705003484665
io1gn450vcrjkh7r35gcjyt93tp3u2dadz3k07wzw   robotbp00023     24           active   12                       2289494.821788705003484665
io120au9ra0nffdle04jx2g5gccn6gq8qd4fy03l4   robotest         25                    0                        1871118.444727959406748141
```

### CODE REVIEW

File: ioctl/config/config.go [hunk 1]
```diff
@@ -103,6 +104,9 @@ func init() {
 		ReadConfig.Language = supportedLanguage[0]
 		completeness = false
 	}
+	if ReadConfig.FairBankHeight == 0 {
+		ReadConfig.FairBankHeight = 5165641
```
Comment (dustinxie) [inline ioctl/config/config.go:108]: use config.Default.Genesis.Fb?

File: ioctl/cmd/node/nodedelegate.go [hunk 2]
```diff
@@ -101,43 +105,15 @@ func (m *delegatesMessage) String() string {
...
-			return fmt.Sprintf("delegates of upcoming epoch #%d are not determined", epochNum)
-		}
-		aliasLen := 5
-		for _, bp := range m.Delegates {
-			if len(bp.Alias) > aliasLen {
-				aliasLen = len(bp.Alias)
+			"Address", "Name", "Rank", "Alias", "Status", "Blocks", "ProbatedStatus", "Votes"))
+		for i, bp := range m.Delegates {
+			if i == 36 {
+				lines = append(lines, "------------------------------")
```
Comment (yutongp) [inline ioctl/cmd/node/nodedelegate.go:114]: probably just remove this. this make an assumption that 36 is an important number and hard coded here

Comment (yutongp) [reply ioctl/cmd/node/nodedelegate.go:114]: we cannot make this assumption that top 36 is different than others. so this separate line is not necessary

Comment (lzxm160) [reply ioctl/cmd/node/nodedelegate.go:114]: I'll remove it

Comment (dustinxie) [reply ioctl/cmd/node/nodedelegate.go]: add this after active producer list

File: ioctl/config/config.go (hunk 1 above)
Comment (dustinxie) [reply ioctl/config/config.go:108]: also can do this in initConfig() ?

File: ioctl/cmd/node/nodedelegate.go (hunk 2 above)
Comment (lzxm160) [reply ioctl/cmd/node/nodedelegate.go]: In String(),it's diffuclt to check which one is bp or not

**Pull request merged**